src/shared/utils/getDeclarationErrorAddendum.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule getDeclarationErrorAddendum
```

Comment:
I will probably just refactor the code so that things make more sense, rather than trying to maintain the way that it was working before. So we probably don't need this as a separate module.